docs/package-imports.md

```diff
@@ -0,0 +1,41 @@
+An agent that is generated from the AEA framework is a modular system with different skills, connections, and protocols.
+When we create a new agent with the command `aea create my_aea` we create the file structure that looks like the following :
+- connections
+- protocols
+- skills
+- vendor
+- aea-config.yaml
+
+In all the folders except the `vendor`, we can locate the packages we developed. On the other hand, if we download a 
```

Comment:
The `vendor` folder contains the packages from the registry which have been developed by other authors or ourselves. The packages we developed as part of the given AEA project are in the respective `connections`, `protocols`, and `skills` folders.

Reply:
Inside `vendor` the packages are name spaced by author name.